test/dkms_dependencies_test-1.0/dkms.conf

```diff
@@ -3,8 +3,9 @@ PACKAGE_NAME="dkms_dependencies_test"
 PACKAGE_VERSION="1.0"
 BUILT_MODULE_NAME="dkms_dependencies_test"
 
-MAKE="echo ERROR: This module fails to build.; \
-    exit 1"
+BUILD_DEPENDS="dkms_failing_test"
```

Comment:
As mentioned at the top - please introduce this and the other conf in there. No point in adding them (unused) only to be changed immediately after.

That said - huge thanks for the tests. Apart from the small suggestions they're looking great.

Reply:
Done.